BrainPortal/app/views/resource_usage/_resource_usage_table.html.erb

```diff
@@ -102,32 +102,37 @@
   </p>
   <% end %>
 </fieldset>
+
+<p>
+
 <fieldset>
+
   <legend>
     Additional filtering
-    <%= show_hide_toggle "(show/hide)", ".filtration", :class => 'task_help_link', :slide_effect => true, :slide_duration => 'fast'  %>
+    <%= show_hide_toggle "(show/hide)", ".filtering", :class => 'task_help_link', :slide_effect => true, :slide_duration => 'fast'  %>
   </legend>
 
 
   <%
-    is_hidden = (params[:data] && params[:data][:created_at].present?) ||
-                 params[:deleted_items].present?                       ||
-                (params[:positive_file_delta].present? && @maintype =~ /Userfile/) ||
-                (params[:negative_file_delta].present? && @maintype =~ /Userfile/) ?
-                 "" : "hidden"
+    is_hidden = true
+
+    is_hidden = false if (params[:date_range] && params[:date_range][:created_at].present?)
+    is_hidden = false if  params[:deleted_items].present?
+    is_hidden = false if (params[:positive_file_delta].present? && @maintype =~ /Userfile/)
+    is_hidden = false if (params[:negative_file_delta].present? && @maintype =~ /Userfile/)
   %>
 
-  <div class="simple filtration <%= is_hidden %>">
+  <div class="simple filtering <%= is_hidden ? "hidden" : "" %>">
     <%= form_tag({ :action  => :index }, :method => :get) do %>
 
-      <table style="border:0px">
-        <tr  style="border:0px">
-          <td rowspan=3 class="left" style="border:0px">
-            <%= date_range_panel(params[:data] || {}, "data", :date_attributes => {:created_at => "By creation date"}) %>
+      <table style="border: none">
```

Comment:
This table is still wrong... there are not as many TR as your rowspan value for one of your TDs